AK/Random.h

```diff
@@ -17,7 +17,7 @@
 #    include <unistd.h>
 #endif
 
-#if defined(AK_OS_MACOS)
+#if defined(AK_OS_MACOS) || defined(AK_OS_WIN32)
 #    include <sys/random.h>
 #endif
 
```

Comment:
line 31 below is `#elif defined(__unix__) or defined(AK_OS_MACOS)` doesn't that need to change to match line 20. So `#elif defined(__unix__) || defined(AK_OS_MACOS) || defined(AK_OS_WIN32)`